tests/clusterclient.c

```diff
@@ -54,13 +54,16 @@ void eventCallback(const redisClusterContext *cc, int event, void *privdata) {
 }
 
 int main(int argc, char **argv) {
+    int use_cluster_slots = 1;
     int show_events = 0;
     int send_to_all = 0;
 
     int argindex;
     for (argindex = 1; argindex < argc && argv[argindex][0] == '-';
          argindex++) {
-        if (strcmp(argv[argindex], "--events") == 0) {
+        if (strcmp(argv[argindex], "--use-cluster-nodes") == 0) {
+            use_cluster_slots = 0;
+        } else if (strcmp(argv[argindex], "--events") == 0) {
```

Comment:
Nit: If you would add "--use-cluster-nodes" in an else-if (after "--events"), you would get a smaller diff.

Reply:
I used the same order as in clusterclient_async to have their main() be more alike at start. But they differ in some parts anyway..